assets/CMakeLists.txt

```diff
@@ -4,14 +4,18 @@ if(UNIX AND NOT APPLE)
     include(KDEInstallDirs)
 
     foreach(_i 16 24 32 64 128)
-        install(FILES "freedesktop/appicons/${_i}/zeal.png"
+        install(FILES "freedesktop/appicons/${_i}/org.zealdocs.Zeal.png"
             DESTINATION "${KDE_INSTALL_ICONDIR}/hicolor/${_i}x${_i}/apps"
             # TODO: Use `RENAME zeal.png` and get rid of subdirectories.
         )
     endforeach()
 
-    # TODO: Generate via zeal.desktop.in.
-    install(FILES "freedesktop/zeal.desktop"
+    # TODO: Generate via org.zealdocs.Zeal.desktop.in.
+    install(FILES "freedesktop/org.zealdocs.Zeal.desktop"
```

Comment:
Is this compatible with all DEs?

Reply:
Yes, DEs generally don't care about filename.

Reply:
I've been trying to find any specification telling to use reverse URL naming, but failed so far.

Here's what [GNOME docs](https://developer.gnome.org/icon-naming-spec/#guidelines) say:
> An application must either use a generic application icon name provided by this specification, or install an icon named the same as the executable for running the application.

Any other sources? I kind of like the URL-based convention, but I have never seen any project using it.